prow/images/test-runner/Dockerfile

```diff
@@ -13,10 +13,9 @@
 # limitations under the License.
 
 FROM gcr.io/k8s-testimages/kubekins-e2e:v20190729-351ea95-master
-LABEL maintainer "Adriano Cunha <adrcunha@google.com>"
+LABEL maintainer "Vincent Demeester <vdemeest@redhat.com>"
```

Comment:
NIT: I guess we could say "The Tekton Authors"

Reply:
Using `tekton-dev` google groups for the email ? that could work :stuck_out_tongue_closed_eyes: